modules/nw-ovn-kubernetes-running-network-tools.adoc

```diff
@@ -0,0 +1,223 @@
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-running-network-tools_{context}"]
+= Running network-tools
+
+Get information about the logical switches and routers by running `network-tools`.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You have installed `network-tools` on local host.
+
+.Procedure
+
+. List the routers by running the following command:
++
+[source,terminal]
+----
+$ ./debug-scripts/network-tools ovn-db-run-command ovn-nbctl lr-list
+----
++
+.Example output
++
+[source,terminal]
+----
+Leader pod is ovnkube-master-vslqm
+5351ddd1-f181-4e77-afc6-b48b0a9df953 (GR_helix13.lab.eng.tlv2.redhat.com)
+ccf9349e-1948-4df8-954e-39fb0c2d4d06 (GR_helix14.lab.eng.tlv2.redhat.com)
+e426b918-75a8-4220-9e76-20b7758f92b7 (GR_hlxcl7-master-0.hlxcl7.lab.eng.tlv2.redhat.com)
+dded77c8-0cc3-4b99-8420-56cd2ae6a840 (GR_hlxcl7-master-1.hlxcl7.lab.eng.tlv2.redhat.com)
+4f6747e6-e7ba-4e0c-8dcd-94c8efa51798 (GR_hlxcl7-master-2.hlxcl7.lab.eng.tlv2.redhat.com)
+52232654-336e-4952-98b9-0b8601e370b4 (ovn_cluster_router)
+----
+
+. List the localnet ports by running the following command:
++
+[source,terminal]
+----
+$ ./debug-scripts/network-tools ovn-db-run-command \
+ovn-sbctl find Port_Binding type=localnet
+----
++
+.Example output
++
+[source,terminal]
+----
+Leader pod is ovnkube-master-vslqm
+_uuid               : 3de79191-cca8-4c28-be5a-a228f0f9ebfc
+additional_chassis  : []
+additional_encap    : []
+chassis             : []
+datapath            : 3f1a4928-7ff5-471f-9092-fe5f5c67d15c
+encap               : []
+external_ids        : {}
+gateway_chassis     : []
+ha_chassis_group    : []
+logical_port        : br-ex_helix13.lab.eng.tlv2.redhat.com
+mac                 : [unknown]
+nat_addresses       : []
+options             : {network_name=physnet}
+parent_port         : []
+port_security       : []
+requested_additional_chassis: []
+requested_chassis   : []
+tag                 : []
+tunnel_key          : 2
+type                : localnet
+up                  : false
+virtual_parent      : []
+
+_uuid               : dbe21daf-9594-4849-b8f0-5efbfa09a455
+additional_chassis  : []
+additional_encap    : []
+chassis             : []
+datapath            : db2a6067-fe7c-4d11-95a7-ff2321329e11
+encap               : []
+external_ids        : {}
+gateway_chassis     : []
+ha_chassis_group    : []
+logical_port        : br-ex_hlxcl7-master-2.hlxcl7.lab.eng.tlv2.redhat.com
+mac                 : [unknown]
+nat_addresses       : []
+options             : {network_name=physnet}
+parent_port         : []
+port_security       : []
+requested_additional_chassis: []
+requested_chassis   : []
+tag                 : []
+tunnel_key          : 2
+type                : localnet
+up                  : false
+virtual_parent      : []
+
+[...]
+----
+
+. List the l3gateway ports by running the following command:
```

Comment:
is l3gateway the common way to refer to this? Consider the following:
```suggestion
. List the L3 gateway ports by running the following command:
```
or maybe it makes more sense to keep the original, but put it in backticks since it seems to be referring to a command option:
```suggestion
. List the `l3gateway` ports by running the following command:
```